scripts/qa_crowbarsetup.sh

```diff
@@ -3151,9 +3155,12 @@ function onadmin_proposal()
         done
     fi
     local proposal
-    for proposal in nfs_client pacemaker database rabbitmq keystone swift ceph glance cinder neutron nova `horizon_barclamp` ceilometer heat manila trove magnum tempest; do
+    for proposal in nfs_client pacemaker database rabbitmq keystone swift ceph glance cinder neutron nova `horizon_barclamp` ceilometer heat manila trove tempest; do
         deploy_single_proposal $proposal
     done
+    if [[ $want_magnum ]]; then
```

Comment:
we usually skip the proposal in the ```do_single_proposal``` function. Can you move it there and readd ```magnum``` to the loop at https://github.com/SUSE-Cloud/automation/pull/970/commits/56fc8812cbd8a5fcfa787230d501a01456064a58#diff-c202af9895fd7be9224b15a93a00b0ffR3158 ?